server/api/min_resolved_ts.go

```diff
@@ -38,17 +40,18 @@ func newMinResolvedTSHandler(svr *server.Server, rd *render.Render) *minResolved
 
 // NOTE: This type is exported by HTTP API. Please pay more attention when modifying it.
 type minResolvedTS struct {
-	IsRealTime      bool              `json:"is_real_time,omitempty"`
-	MinResolvedTS   uint64            `json:"min_resolved_ts"`
-	PersistInterval typeutil.Duration `json:"persist_interval,omitempty"`
+	IsRealTime         bool              `json:"is_real_time,omitempty"`
+	MinResolvedTS      uint64            `json:"min_resolved_ts"`
+	PersistInterval    typeutil.Duration `json:"persist_interval,omitempty"`
+	StoreMinResolvedTS map[uint64]uint64 `json:"store_min_resolved_ts"`
```

Comment:
it will return all stores, if there is many stores, it will be difficult to read

Reply:
Empty if no store list is specified.